site/src/components/DividerWithText/DividerWithText.tsx

```diff
@@ -0,0 +1,32 @@
+import { makeStyles } from "@material-ui/core/styles"
+import { FC, PropsWithChildren } from "react"
+
+export const DividerWithText: FC<PropsWithChildren> = ({ children }) => {
+  const classes = useStyles()
+  return (
+    <div className={classes.container}>
+      <div className={classes.border} />
+      <span className={classes.content}>{children}</span>
+      <div className={classes.border} />
+    </div>
+  )
+}
+
+const useStyles = makeStyles((theme) => ({
+  container: {
+    display: "flex",
+    alignItems: "center",
+  },
+  border: {
+    borderBottom: `2px solid ${theme.palette.divider}`,
+    width: "100%",
+  },
+  content: {
+    paddingTop: theme.spacing(0.5),
+    paddingBottom: theme.spacing(0.5),
+    paddingRight: theme.spacing(2),
+    paddingLeft: theme.spacing(2),
+    fontSize: theme.typography.h5.fontSize,
+    color: theme.palette.text.secondary,
+  },
+}))
```

Comment:
This is a fun component! I might colocate it within the `site/src/pages/DeploySettingsPage/LicensesSettingsPage` folder as it's only used on that page at the moment.

Reply:
cool! fixed! :)